# HG changeset patch
# User David Grundberg <davidg@cs.umu.se>
# Date 1257280480 18000
#      Tue Nov 03 15:34:40 2009 -0500
# Node ID 6e6d8ea324bc28e757eeabddb6b2f1f5a37fcbd7
# Parent  266eae13dcf30da33d9c50f64cc2a4e2209c5c69
Add property FormatSpacing to root

diff --git a/src/ChangeLog b/src/ChangeLog
--- a/src/ChangeLog
+++ b/src/ChangeLog
@@ -1,8 +1,12 @@
+2009-11-03  David Grundberg  <davidg@cs.umu.se>
+
+	* graphics.h.in (root_figure): Add FormatSpacing property.
+
 2009-11-03  David Grundberg  <davidg@cs.umu.se>
 
 	* ov-class.cc (octave_class::size): Allow and ignore extra outargs
 	from user-defined size methods.
 
 2009-11-03  David Grundberg  <davidg@cs.umu.se>
 
 	* ov-class.cc (make_idx_args): Call user-defined subsref/subsasgn
diff --git a/src/graphics.h.in b/src/graphics.h.in
--- a/src/graphics.h.in
+++ b/src/graphics.h.in
@@ -2205,16 +2205,17 @@ public:
 
     BEGIN_PROPERTIES (root_figure, root)
       handle_property currentfigure S , graphics_handle ()
       handle_property callbackobject Sr , graphics_handle ()
       double_property screendepth r , default_screendepth ()
       array_property screensize r , default_screensize ()
       double_property screenpixelsperinch r , default_screenpixelsperinch ()
       radio_property units U , "inches|centimeters|normalized|points|{pixels}"
+      radio_property formatspacing , "{loose}|compact"
       bool_property showhiddenhandles , "off"
     END_PROPERTIES
 
   private:
     std::list<graphics_handle> cbo_stack;
   };
 
 private:
